### ISSUE TITLE
Rolling up errors out as  "Promise is not defined"

### ISSUE THREAD

**Issue by ayushmad** [rollup/rollup#254]
This seems a really cool project and I was looking forward to use it. However I hit a small bug while using it.
When I use the following command to rollup.
```javascript
$ rollup -i src/index.js -o newTest.js
Promise is not defined
ReferenceError: Promise is not defined
    at Object.write (/usr/local/lib/node_modules/rollup/Volumes/Storage/www/ROLLUP/rollup/src/rollup.js:53:12)
    at /usr/local/lib/node_modules/rollup/bin/runRollup.js:110:18
    at tryCatch (/usr/local/lib/node_modules/rollup/Volumes/Storage/www/ROLLUP/rollup/node_modules/es6-promise/lib/es6-promise/-internal.js:182:12)
    at invokeCallback (/usr/local/lib/node_modules/rollup/Volumes/Storage/www/ROLLUP/rollup/node_modules/es6-promise/lib/es6-promise/-internal.js:194:13)
    at publish (/usr/local/lib/node_modules/rollup/Volumes/Storage/www/ROLLUP/rollup/node_modules/es6-promise/lib/es6-promise/-internal.js:164:16)
    at flush (/usr/local/lib/node_modules/rollup/Volumes/Storage/www/ROLLUP/rollup/node_modules/es6-promise/lib/es6-promise/asap.js:86:25)
    at process._tickCallback (node.js:448:13)
    at Function.Module.runMain (module.js:499:11)
    at startup (node.js:119:16)
    at node.js:935:3
```
However if I use a slightly different format of command it does *rollup* perfectly.

```javascript
$ rollup --format umd  --name dq -- src/index.js > dq.js
```

This might be a really small bug with the command line parameter. I just wanted you to be aware of this.
Again thank you for creating this tool.

**Comment by Rich-Harris** [comment]
Ah, looks like we forgot to import Promise in [src/rollup.js](https://github.com/rollup/rollup/blob/e111c5d8dc5d826fed1341870bbff6b22b9a8459/src/rollup.js#L53), so it's falling back to `global.Promise`. We'll fix that for the next version.

In the meantime, are you using an old version of Node.js (`node -v` to find out)? If you're running 0.12 or lower then `Promise` isn't available, but if you install an up-to-date version from https://nodejs.org/en/ then it should work.

**Comment by ayushmad** [comment]
You are right. My node version is 0.10.40
```javascript
$ node -v
v0.10.40
```
I updated my node version to 5.0.0 and everything works great now.
Thanks for the help.

**Comment by TrySound** [comment]
@Rich-Harris node 0.12 supports Promises

**Comment by ayushmad** [comment]
Verified the fix works in node 0.10.40. 
Thanks for the fix.
